$OpenBSD: patch-gst-libs_ext_libav_libavcodec_Makefile,v 1.2 2014/10/07 16:06:09 ajacoutot Exp $

libavcodec.a(log2_tab.o)(.rodata+0x0): multiple definition of `ff_log2_tab'
libavformat.a(log2_tab.o)(.rodata+0x0): first defined here

--- gst-libs/ext/libav/libavcodec/Makefile.orig	Wed Sep 24 10:55:09 2014
+++ gst-libs/ext/libav/libavcodec/Makefile	Tue Oct  7 17:22:20 2014
@@ -18,7 +18,6 @@ OBJS = allcodecs.o                                    
        codec_desc.o                                                     \
        fmtconvert.o                                                     \
        imgconvert.o                                                     \
-       log2_tab.o                                                       \
        mathtables.o                                                     \
        options.o                                                        \
        parser.o                                                         \
